addons/controllers/clusterbootstrap_handlers.go

```diff
@@ -76,6 +79,52 @@ func (r *ClusterBootstrapReconciler) ClusterBootstrapToClusters(o client.Object)
 	return []ctrl.Request{{NamespacedName: client.ObjectKeyFromObject(cluster)}}
 }
 
+// SecretsToClusters is the Map Function for watching Secrets that filters the events on
+// objects of type secret and returns requests for reconcile if required
+func (r *ClusterBootstrapReconciler) SecretsToClusters(o client.Object) []ctrl.Request {
+	secret, ok := o.(*corev1.Secret)
+	if !ok {
+		r.Log.Error(errors.New("invalid type"),
+			"Expected to receive Secret resource",
+			"actualType", fmt.Sprintf("%T", o))
+		return nil
+	}
+
+	log := r.Log.WithValues(constants.SecretNameLogKey, secret.Name)
+	log.V(4).Info("Mapping secrets to cluster")
```

Comment:
question: could you elaborate a little bit on why we want to have L93-94 instead of using `r.Log.Info()` ?

Reply:
I am not sure abt the exact details, but I tried to keep it similar to other 2 map functions of watches => ClusterBootstrapToClusters, TKRToClusters